cloudformation/template.yml

```diff
@@ -24,11 +24,14 @@ Metadata:
           - MaximumVCPUs
       -
         Label:
-          default: Docker Container Configuration
+          default: Docker Image Configuration
         Parameters:
           - RepositoryName
-          - ImageName
```

Comment:
As you'll see lower down, `ImageName` isn't actually used in the ECR repo URI, so I removed it from the parameter list.